### PR TITLE
Allow jna and jna-platform artifacts to be not productized.

### PR DESCRIPTION
**Pull request by jedla97**
As the title say. Tested locally and these artifact not appearing in error output.

### CODE REVIEW

File: testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistProductBomJars.java [hunk 1]
```diff
@@ -13,6 +13,8 @@ public enum WhitelistProductBomJars {
             "com.nimbusds.oauth2-oidc-sdk",
             "com.nimbusds.content-type",
             "io.github.crac",
+            "net.java.dev.jna.jna",
```
Comment (mjurc) [inline testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistProductBomJars.java:16]: Can you add a comment about where these deps come from?

**Review by mjurc — CHANGES_REQUESTED** [review]
Otherwise LGTM - thanks for the contribution.

File: testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistProductBomJars.java (hunk 1 above)
Comment (jedla97) [reply testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistProductBomJars.java:17]: Updated

**Comment by jedla97** [comment]
This one failing test appearing also in nightly. So this PR not causing it.

**Review by mjurc — APPROVED** [review]
Thanks for the contribution.

**Pull request merged**